plugins/transforms/selectvalues/src/test/java/org/apache/hop/pipeline/transforms/selectvalues/SelectValuesMetaTest.java

```diff
@@ -40,9 +40,13 @@
 public class SelectValuesMetaTest {
```

Comment:
This class should be ok. I've removed a few tests that wouldn't make any sense with the current structure.
Well, the remaining ones maybe neither, but less than the evicted ones. 😄 

However  the focus of this class was already the SelectFields tab, while the Remove and ChangeMetadata part was skipped over completely.